openapi.yaml

```diff
@@ -596,11 +604,14 @@ components:
           type: string
           description: Incremented value for each file for RDFI's.
           example: "0"
+          nullable: true
       required:
         - immediateOrigin
         - immediateOriginName
         - immediateDestination
         - immediateDestinationName
+        - fileCreationTime
```

Comment:
@adamdecaf per the openapi spec: `Note that an empty string "" is a valid string unless minLength or pattern is specified.`

So would it make sense to add this field to `required` since we don't specify `omitempty` on it?

ref: https://swagger.io/docs/specification/data-models/data-types/

Reply:
^^ nvm, ignore the comment above, I'm going to add the `pattern` specifier to fields like this

Reply:
Do we set a value if that string is empty? I would think the library would populate it on `Create()` if empty.

Reply:
For `FIleCreationTime`, it'll stay empty

Reply:
Oh it's because when we write the FileHeader an empty string is written as the current time. It's fine if that is left out of a request. We'll populate the correct value when constructing the file.

https://github.com/moov-io/ach/blob/master/fileHeader.go#L339

Reply:
oh i see, thanks for the link!